packages/ckeditor5-dev-env/tests/translations/upload.js

```diff
@@ -10,67 +10,82 @@
 const path = require( 'path' );
 const sinon = require( 'sinon' );
 const mockery = require( 'mockery' );
-const transifexService = require( '../../lib/translations/transifex-service' );
-const fs = require( 'fs' );
 
 describe( 'upload', () => {
-	let sandbox;
+	const sandbox = sinon.sandbox.create();
 	let upload;
+	let packageNames;
+	let stubs;
 
 	beforeEach( () => {
-		sandbox = sinon.sandbox.create();
+		mockery.enable( {
+			useCleanCache: true,
+			warnOnReplace: false,
+			warnOnUnregistered: false,
+		} );
+
+		stubs = {
+			logger: {
+				info: sandbox.stub(),
+				warning: sandbox.stub(),
+				error: err => {
+					throw err;
```

Comment:
This is a little different situation. Because of the promise catch at https://github.com/ckeditor/ckeditor5-dev/blob/15ccf2b3de21815e95918cf2b0a7037c2ac90da9/packages/ckeditor5-dev-env/lib/translations/upload.js Now I'm not sure if this catch should be there, because it complicates the tests. But from the other hand this function should log the error. Maybe I can throw the caught error and handle it outside this function.